api/v1beta1/ovndbcluster_types.go

```diff
@@ -28,6 +28,13 @@ const (
 	NBDBType = "NB"
 	// SBDBType - Southbound database type
 	SBDBType = "SB"
+
+	// Container image fall-back defaults
+
+	// OvnNbContainerImage is the fall-back container image for OVNDBCluster NB
+	OvnNbContainerImage = "quay.io/podified-antelope-centos9/openstack-ovn-nb-db-server:current-podified"
```

Comment:
would be good to name these as OvnNBContainerImage for consistency

Reply:
Done